.travis.yml

```diff
@@ -6,12 +6,16 @@ language: php
 matrix:
   fast_finish: true
   include:
-    - env: DB=mysql; MW=REL1_28; TYPE=coverage
+    - env: DB=mysql; MW=REL1_29; TYPE=coverage
       php: 5.6
     - env: DB=sqlite; MW=REL1_27; SITELANG=ja
       php: 5.6
     - env: DB=mysql; MW=REL1_30; PHPUNIT=4.8.*
-      php: 7
+      php: 7.1
+    - env: DB=mysql; MW=REL1_31; PHPUNIT=6.5.*
+      php: 7.1
+    - env: DB=sqlite; MW=master; PHPUNIT=7.3.*
```

Comment:
PHPUnit 7.3 is a bit too optimistic because MW core just fails with:

```
PHP Fatal error:  Declaration of MediaWikiPHPUnitCommand::createRunner() must be compatible with PHPUnit\TextUI\Command::createRunner(): PHPUnit\TextUI\TestRunner in /home/travis/build/SemanticMediaWiki/mw/tests/phpunit/MediaWikiPHPUnitCommand.php on line 3
PHP Stack trace:
PHP   1. {main}() /home/travis/build/SemanticMediaWiki/mw/tests/phpunit/phpunit.php:0
PHP   2. require() /home/travis/build/SemanticMediaWiki/mw/tests/phpunit/phpunit.php:129
PHP   3. PHPUnitMaintClass->execute() /home/travis/build/SemanticMediaWiki/mw/maintenance/doMaintenance.php:94
PHP   4. spl_autoload_call() /home/travis/build/SemanticMediaWiki/mw/tests/phpunit/phpunit.php:89
PHP   5. AutoLoader::autoload() /home/travis/build/SemanticMediaWiki/mw/tests/phpunit/phpunit.php:89
PHP   6. require() /home/travis/build/SemanticMediaWiki/mw/includes/AutoLoader.php:109
Fatal error: Declaration of MediaWikiPHPUnitCommand::createRunner() must be compatible with PHPUnit\TextUI\Command::createRunner(): PHPUnit\TextUI\TestRunner in /home/travis/build/SemanticMediaWiki/mw/tests/phpunit/MediaWikiPHPUnitCommand.php on line 3
Call Stack:
    0.0002     428320   1. {main}() /home/travis/build/SemanticMediaWiki/mw/tests/phpunit/phpunit.php:0
    0.0025     726592   2. require('/home/travis/build/SemanticMediaWiki/mw/maintenance/doMaintenance.php') /home/travis/build/SemanticMediaWiki/mw/tests/phpunit/phpunit.php:129
    0.1168   15180920   3. PHPUnitMaintClass->execute() /home/travis/build/SemanticMediaWiki/mw/maintenance/doMaintenance.php:94
    0.1210   15782296   4. spl_autoload_call() /home/travis/build/SemanticMediaWiki/mw/tests/phpunit/phpunit.php:89
    0.1210   15782344   5. AutoLoader::autoload() /home/travis/build/SemanticMediaWiki/mw/tests/phpunit/phpunit.php:89
    0.1211   15787496   6. require('/home/travis/build/SemanticMediaWiki/mw/tests/phpunit/MediaWikiPHPUnitCommand.php') /home/travis/build/SemanticMediaWiki/mw/includes/AutoLoader.php:109
```